Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -10,17 +10,18 @@ button.webButton
   cursor: pointer;
 }
 
-button.webButton:hover,
-button.webButton:focus-within
+button.webButton:hover
 {
   background-color: var(--remotion-themed-background-color-hover);
   border: var(--remotion-themed-border-hover);
   box-shadow: var(--remotion-themed-box-shadow-hover);
 }
 
-button.webButton:focus-within
+button.webButton:focus
```

Comment:
button is focusable. there is never going to be a child element that gets the focus.

focus-within is only needed when the acutally focused target and the highlight effects aren't on the same element.